api/docs/release.dox

```diff
@@ -145,6 +145,9 @@ compatibility changes:
    expression and code relying on this needs to be rewritten.
    DynamoRIO_NUM_SIMD_SLOTS_COMPATIBILITY is set automatically if clients target
    version 7.1.0 or earlier.
+ - Added the type dr_zmm_t.
+ - Added the type dr_opmask_t.
```

Comment:
As before in prior PR's, please use the `#` prefix to get a link for these types on these two lines 148 and 149.

Reply:
I keep forgetting those. Done. Will amend in small PR.

Reply:
As suggested before, please look through the diff in a self-review before posting as a PR.  That should be catching things like this, accidentally left-in changes seen in other PR's, etc.